shardingsphere-infra/shardingsphere-infra-common/src/main/java/org/apache/shardingsphere/infra/distsql/exception/rule/RequiredAlgorithmMissedException.java

```diff
@@ -33,4 +33,8 @@ public RequiredAlgorithmMissedException(final String schemaName) {
     public RequiredAlgorithmMissedException(final String schemaName, final Collection<String> algorithmNames) {
         super(1115, String.format("Sharding algorithms `%s` do not exist in schema `%s`.", algorithmNames, schemaName));
```

Comment:
what does 1115 mean . is there any constant or enum to describe this ?

Reply:
This is `errorCode`, the same exception class `errorCode` is the same, 1115 is used directly in the current class, so I keep the same, but I did not find an explanation for 1115.